app/src/main/java/com/example/okuzminykh/assignmentfirstweek/AnimationFragment.kt

```diff
@@ -1,22 +1,101 @@
 package com.example.okuzminykh.assignmentfirstweek
 
 import android.os.Bundle
-import androidx.fragment.app.Fragment
+import android.os.Handler
+import android.os.Looper
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.okuzminykh.assignmentfirstweek.databinding.FragmentAnimationBinding
+
+const val SECONDS = "seconds"
+const val RUNNING = "running"
+const val WAS_RUNNING = "was running"
 
 class AnimationFragment : Fragment() {
+    var seconds = 5.0
+    var running = true
+    private var wasRunning = true
+    private lateinit var binding: FragmentAnimationBinding
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-    }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_animation, container, false)
+    ): View {
+        binding = FragmentAnimationBinding.inflate(layoutInflater, container, false)
+        binding.root.setOnClickListener { startAndStopTimer() }
```

Comment:
use binding for onclick, create a viewmodel and pass it into binding as variable